docs/source/getting_started/teams.rst

```diff
@@ -0,0 +1,132 @@
+========
+Teams
+========
+
+Teams overview
```

Comment:
This line should either be fleshed out into a sentence, or else dropped. Either makes sense to me.